### ISSUE TITLE
Security vulnerability in jackson-databind@2.11.0

### ISSUE THREAD

**Issue by equanz** [apache/bookkeeper#3134]
**BUG REPORT**

***Describe the bug***

`OWASP Dependency Check` job detects the vulnerability in `com.fasterxml.jackson.core/jackson-databind@2.11.0`.

```
One or more dependencies were identified with known vulnerabilities in bookkeeper:

jackson-databind-2.11.0.jar (pkg:maven/com.fasterxml.jackson.core/jackson-databind@2.11.0, cpe:2.3:a:fasterxml:jackson-databind:2.11.0:*:*:*:*:*:*:*) : CVE-2020-36518
```

https://nvd.nist.gov/vuln/detail/CVE-2020-36518

**Comment by nicoloboschi** [comment]
We have to wait until 2.13.2 is released 
https://github.com/FasterXML/jackson-databind/issues/2816#issuecomment-1073483255

**Comment by equanz** [comment]
@marcospds 
> will a fix be created for 2.11.x? Like a 2.11.5 or 2.11.4.1 version?

Which does your comment mean, "*Will a fix be created for bookkeeper v2.11.x*" or "*Will a fix be created for jackson-databind v2.11.x*" ?

**Comment by eolivelli** [comment]
See here https://github.com/apache/bookkeeper/pull/3140
